dotnet/src/SemanticKernel/Reliability/RetryWithDelay.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using Microsoft.SemanticKernel.AI;
+using Microsoft.SemanticKernel.Diagnostics;
+
+namespace Microsoft.SemanticKernel.Reliability;
+
+/// <summary>
+/// A retry mechanism that retries once after a delay, using RetryAfter header if available.
+/// </summary>
+public class RetryWithDelay : IRetryMechanism
+{
+    private int RetryCount { get; } = 2;
+    private TimeSpan RetryDelay { get; } = TimeSpan.FromSeconds(2);
```

Comment:
usually we don't use props for private values, because refactoring doesn't break clients, plus there's no extra logic here

Reply:
Yup, fixed this!